### ISSUE TITLE
unify RSS feed elements for directory and file

### ISSUE THREAD

**Issue by vladak** [OpenGrok/OpenGrok#896]
RSS feed `description` field differs for files and directories. For directory it contains list of files changed in given changeset, for file it contains path and changeset ID.

The changeset ID could be displayed in the `title` (maybe with first line of the comments) and the full comments (with newlines) in `description`.

**Comment by vladak** [comment]
Small issue is that `DirectoryHistoryReader` class does not populate the history entries with revision IDs but that can be fixed by extending the giant hash structure used in the class.

**Comment by vladak** [comment]
The other issue is how to display newlines in the `description` field. Currently the `htmlize` method converts newlines to `br` element automatically. Even with system newlines RSS readers might have troubles displaying them, need to check RSS spec.

**Comment by vladak** [comment]
For directories one entry will look like this:

```
<item>
        <title>df0f3123 - initial debian packaging (draft)</title>
        <link>http://localhost:8080/source/history/OpenGrok/#df0f3123</link>
        <description>
            initial debian packaging (draft)

            List of files:
            
            /OpenGrok/debian/README.Debian
            /OpenGrok/debian/ant.properties
            /OpenGrok/debian/changelog
            /OpenGrok/debian/compat
            /OpenGrok/debian/opengrok.docs
            /OpenGrok/debian/opengrok.install
            /OpenGrok/debian/rules
        </description>
        <pubDate>Tue, 31 May 2011 16:00:00 +0200</pubDate>
        <dc:creator>Lubos Kosco &lt;Lubos.Kosco@Oracle.COM&gt;</dc:creator>
    </item>
```

and for files like this:

```
<item>
        <title>83439b4e - introduce OPENGROK_DERBY_URL for selecting JavaDB instance</title>
        <link>http://localhost:8080/source/history/OpenGrok/README.txt#83439b4e</link>
        <description>
            introduce OPENGROK_DERBY_URL for selecting JavaDB instance
                fixes #656
                fixes #529

            List of files:
            /OpenGrok/README.txt
        </description>
        <pubDate>Thu, 08 Aug 2013 17:00:26 +0200</pubDate>
        <dc:creator>Vladimir Kotal &lt;Vladimir.Kotal@Oracle.COM&gt;</dc:creator>
    </item>
```
